### PR TITLE
Minor Improvements

### PR DESCRIPTION
**Pull request by sanssecours**

### CODE REVIEW

File: scripts/docker/debian/buster/doc.Dockerfile [hunk 1]
```diff
@@ -18,6 +18,8 @@ RUN apt-get -y install \
         texlive-latex-recommended \
         texlive-latex-extra \
         texlive-fonts-recommended \
+        texlive-fonts-extra \
+        texlive-math-extra \
```
Comment (mpranj) [inline scripts/docker/debian/buster/doc.Dockerfile:22]:
```suggestion
        texlive-science \
```

File: scripts/docker/debian/stretch/doc.Dockerfile [hunk 2]
```diff
@@ -19,6 +19,8 @@ RUN apt-get -y install \
         texlive-latex-recommended \
         texlive-latex-extra \
         texlive-fonts-recommended \
+        texlive-fonts-extra \
+        texlive-math-extra \
```
Comment (mpranj) [inline scripts/docker/debian/stretch/doc.Dockerfile:23]:
```suggestion
        texlive-science \
```

File: doc/news/_preparation_next_release.md [hunk 3]
```diff
@@ -120,13 +120,23 @@ you up to date with the multi-language support provided by Elektra.
...
+
 - <<TODO>>
 
 ## Documentation
 
 - Describe hierarchy and limitations of [hosts plugin](https://www.libelektra.org/plugins/hosts). _(Markus Raab)_
-- <<TODO>>
+- The Doxygen PDF documentation now also requires the packages
+  - `stix` (part of `texlive-fonts-extra`) and
+  - `stmaryrd` (part of `texlive-math-extra`). _(René Schwaiger)_
```
Comment (mpranj) [inline doc/news/_preparation_next_release.md:139]:
```suggestion
  - `stmaryrd` (part of `texlive-science` or `texlive-math-extra`). _(René Schwaiger)_
```

**Comment by mpranj** [comment]
Thank you so much for reviewing everything and providing these fixes!

File: scripts/docker/debian/buster/doc.Dockerfile (hunk 1 above)
Comment (sanssecours) [reply scripts/docker/debian/buster/doc.Dockerfile:22]: Thank you for fixing these problems 💖.

**Review by markus2330 — APPROVED** [review]
Thank you so much for this improvement!

File: doc/COMPILE.md [hunk 4]
```diff
@@ -76,7 +76,10 @@ gem install ronn
 To build PDF documentation you need `pdflatex` with
 
 ```sh
-apt-get install pdflatex texlive-fonts-recommended texlive-latex-recommended texlive-latex-extra
+apt-get install pdflatex \
+        texlive-fonts-recommended texlive-fonts-extra \
+        texlive-latex-recommended texlive-latex-extra \
+        texlive-math-extra
```
Comment (mpranj) [inline doc/COMPILE.md:82]:
```suggestion
        texlive-science
```

**Comment by mpranj** [comment]
Thank you! 👍

**Pull request merged**